docs/deployed-contracts/sepolia.md

```diff
@@ -100,6 +100,21 @@ and permission-based [validator set](https://github.com/eth-clients/sepolia/issu
   - L2LidoGateway
   - L2WstETHToken
 
+### Mode
+
+##### Ethereum part
+
+- L1ERC20TokenBridge: [`0xD847780c8808fc99727001C43E6C17A198b39729`](https://sepolia.etherscan.io/address/0xD847780c8808fc99727001C43E6C17A198b39729) (proxy)
+- L1ERC20TokenBridge: [`0x514Aa1d41d840af7572509D75354A8612594eB30`](https://sepolia.etherscan.io/address/0x514Aa1d41d840af7572509D75354A8612594eB30) (impl)
+
+##### Mode part
+
+- WstETH ERC20Bridged: [`0x953757Db814392642e614DaBba794Ef4381D8292`](https://sepolia.explorer.mode.network/address/0x953757Db814392642e614DaBba794Ef4381D8292) (proxy)
```

Comment:
should be [0x2C937931B5d544E3dff1d9D78c80d9772B55837A](https://sepolia.explorer.mode.network/address/0x2C937931B5d544E3dff1d9D78c80d9772B55837A)